README.md

```
@@ -0,0 +1,91 @@
+> 
+
+| Type | Emoji | Description |
+| --- | --- | --- |
+| feat | ✨ (sparkles) | 새로운 기능 추가 |
+| fix | 🐛 (bug) | 기존 기능 및 버그 수정 |
+| docs | 📝 (memo) | 문서 수정 |
+| style | 🎨 (art) | 코드 포맷팅, 세미콜론 누락, 코드 변경이 없는 경우 |
+| refactor | ♻️ (recycle) | 코드 리팩토링(주석 제거도 포함) |
+| test | ✅ (white_check_mark) | 테스트 코드, 리팩토링 테스트 코드 추가 |
+| chore | 🔧 (wrench) | 빌드 업무 수정, 패키지 매니저 수정 |
+| wip | 🚧 (construction) | 완료되지 않은 작업 임시 커밋
```

markdownlint-cli2 (0.18.1)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)